plugins/obs-qsv11/CMakeLists.txt

```diff
@@ -2,12 +2,24 @@ cmake_minimum_required(VERSION 3.22...3.25)
 
 legacy_check()
 
-option(ENABLE_QSV11 "Build Intel QSV11 Hardware Encoder." TRUE)
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(i[3-6]86|x86|x64|x86_64|amd64|AMD64)$")
```

Comment:
You cannot rely on `CMAKE_SYSTEM_PROCESSOR` on Windows because the value will not correctly reflect the toolchain selected for MSVC (e.g. it will always be `x86_64` even when building for ARM64 on an Intel-based Windows machine).

As this change mainly affects Flatpak, guard this off with `if(OS_LINUX)` where CMake toolchain files are more commonly used (which set `CMAKE_SYSTEM_PROCESSOR` accordingly).